about/policy/govt-urls.md

```diff
@@ -1,34 +1,88 @@
 ---
 layout: page
-title: Government-Managed Domains Outside the .Gov and .Mil Top Level Domains
-last_modified_at: March 2, 2019
+title: Government Domains Outside .Gov and .Mil
+last_modified_at: March 11, 2022
 redirect_from:
   - /developer/govt-urls.html
   - /developer/govturls.html
 sidenav: about_us
-date: March 2, 2019
+date: March 11, 2022
 tags: seo policy government-urls
 category: developer
 ---
 ## Overview
 
-As the U.S. government's official web portal, [USA.gov](https://www.usa.gov/) searches across all federal, state, local, tribal, and territorial government websites. Most government websites end in .gov or .mil, but many end in .com, .org, .edu, or other top-level domains.
+Most U.S. government websites end in .gov or .mil, but some end in .com, .org, .net, or other top-level domains. To enhance discoverability and public trust, we maintain a [list](https://github.com/GSA/govt-urls/) of known government domains that *don't* end in .gov or .mil. Federal executive branch agencies must ensure their non-.gov or .mil domains are on the list. Read more in the [Federal Zero Trust Strategy](https://zerotrust.cyber.gov/federal-zero-trust-strategy/#4-safely-making-applications-internet-accessible) and [Policies for Federal Agency Public Websites and Digital Services](https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf#page=11).
 
-In support of USA.gov and [M-17-06 &ndash; Policies for Federal Agency Public Websites and Digital Services](https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf), Search.gov maintains a list of all public government domains that don't end in .gov or .mil.
+The list is used by several U.S. government services, including USA.gov to support their all-of-government search, and the .gov registry to support an inventory of government domains.
+
+Anyone can submit a domain to the list.
```

Comment:
Please update to
Anyone can [submit](#non-gov-domain-collector) a domain to the list.